components/Sidebar.tsx

```diff
@@ -39,7 +39,7 @@ const getDocsPath = [
 const getStartedPath = [
   '/learn/json-schema-examples',
   '/learn/file-system',
-  '/learn/miscellaneous-examples',
+  '/learn/generic-examples',
```

Comment:
I like miscellaneous here.  "Generic" already has specific meaning in programming, and we don't want to imply that these examples are that.

Reply:
I had actually updated the term after discussing with Benjamin on the slack thread ([here](https://json-schema.slack.com/archives/CB3SS5W5B/p1693215510648819?thread_ts=1688030108.211109&cid=CB3SS5W5B)), since miscellaneous and other seemed a bit similar as titles.😅 
I can change it back to Miscellaneous if that seems better, please let me know how to proceed with this.

Reply:
> I can change it back to Miscellaneous

My bad @aku1310. @gregsdennis criteria has more weight here. Please let's rename it back to miscellaneous.